web/app/js/components/Namespace.jsx

```diff
@@ -129,6 +130,7 @@ class Namespaces extends React.Component {
           <div>
             <PageHeader header={`Namespace: ${this.state.ns}`} />
             { noMetrics ? <div>No resources detected.</div> : null}
+            <NetworkGraph deployments={this.state.metrics.deployments} />
```

Comment:
If there aren't metrics, we end up reserving space for an svg that remains empty.
Maybe only show this component if we have metrics:
```
let deploymentsWithMetrics = _.filter(this.state.metrics.deployments, d => !_.isNull(d.requestRate));

{ _.isEmpty(deploymentsWithMetrics) ? null :
            <NetworkGraph deployments={this.state.metrics.deployments} />}
```